README.md

```diff
@@ -1,217 +1,48 @@
-# <span>Keycloak</span> Quickstarts
+# Keycloak Quickstarts
```

Comment:
the `<span>` are used as marker I believe for the productization script.

Reply:
@sebastienblanc damn, thanks for point me to this. I will review the README and also revert this change.

Reply:
@sebastienblanc updated the script and also reverted the tags back.